fem/gridfunc.hpp

```diff
@@ -22,6 +22,7 @@
 #include <limits>
 #include <ostream>
 #include <string>
+#include "../general/kdtree.hpp"
```

Comment:
This should go above, after `config.hpp`, for consistent include ordering (not enforced automatically by astyle)

Reply:
resolved